docs/evaluate/how-it-works.mdx

```diff
@@ -0,0 +1,90 @@
+---
+id: how-it-works
+title: How Temporal Works
+sidebar_label: How It Works
+description: This guide provides a short overview of How Temporal Works.
+toc_max_heading_level: 4
+keywords:
+  - temporal
+  - evaluate-temporal
+  - how-temporal-works
+tags:
+  - temporal
+  - evaluate-temporal
+  - how-temporal-works
+---
+Temporal offers an entirely new way to build scalable and reliable applications.
```

Comment:
by...? Do we want to give a brief explanation?

Reply:
I think I want to keep it short and sweet and hook em in! 🎣